README.md

```diff
@@ -126,11 +126,11 @@ written to the `<JUNIT_DIR>` directory.
 
 To checkout a kernel tree run:
 
-    $ skt --rc <SKTRC> --state --workdir <WORKDIR> -vv \
+    $ skt --rc <SKTRC> --state --workdir <SRCDIR> -vv \
```

Comment:
I think we should keep calling this "workdir". Partly because the option is still named "workdir", and mostly because I cannot vouch at the moment that it is not used for anything else beside just keeping the source.

Reply:
Done. Moved all back to ```workdir``` ```wdir``` and ```WORKDIR```.